data/card_catalogue/story-changing-golds.json

```diff
@@ -8,7 +8,7 @@
         "active": "Become the legal owner of a building of your choice. Existing tenants will be evicted immediately.",
         "passive": null
       },
-      "flavour": "The gang battle of 2485 concluded after the mafioso purchased their fortress and had the police escort them to jail.",
+      "flavour": "Property is not the natural, obvious, and inevitable concept that most people think it is.",
```

Comment:
Avoiding flavour tie ins where possible for cards that I think will make it into the base set (which AFAIK could be any of these)